### PR TITLE
chore: update HOWTO.md

### PR DESCRIPTION
**Pull request by 7flying**
Updates the docs after the new feature in #451 .

Please note that the `plain-di` section has a couple of unexplained environment variables, and that is intended.

### CODE REVIEW

File: HOWTO.md [hunk 1]
```diff
@@ -594,3 +601,136 @@ Please mind how the configuration file must be specifically named (e.g. `-` VS
...
 3. Run the client: `fdo-client-linuxappp`
+
+### Manufacturing client
+
+You can run the `fdo-manufacturing-client` using the [provided
+CLI mode](#cli-mode) or setting up [environment
+variables](#environment-variables-mode).
+
+For both cases, the Manufacturing client will only run if no Device Credentials
+are found in any of the default locations:
```
[the remark this answers is not in the document]
Comment (7flying) [reply HOWTO.md:612]: that is not right

[the remark this answers is not in the document]
Comment (7flying) [reply HOWTO.md:612]: oh boy, yes!

File: HOWTO.md [hunk 2]
```diff
@@ -594,3 +601,136 @@ Please mind how the configuration file must be specifically named (e.g. `-` VS
...
+2. If the Manufacturing server is specifically configured with a
+   `mfg_string_type` set to `MACAddress` in its `diun` configuration section it
+   will ask the device to identify itself with a MAC Address, in order to do so
+   the device must set `DI_MFG_STRING_TYPE_MAC_IFACE` [optional] to a valid
+   interface.
+   
+   Please not that this `DI_MFG_STRING_TYPE_MAC_IFACE` environment variable is
+   *optional* and will only be read if the server requests a `MACAddress`
+   identification mode, but if the Manufacturing server is configured as
+   described above the Device Identification protocol will result in an early
```
Comment (nak3) [inline HOWTO.md:658]: There are some places where `Device Identification` and `Device Initialization` are mixed. But I think here should be `Device Initialization protocol`

File: HOWTO.md [hunk 3]
```diff
@@ -594,3 +601,136 @@ Please mind how the configuration file must be specifically named (e.g. `-` VS
...
+   the specified MAC Address.
+   
+3. Run the client: `fdo-manufacturing-client`.
+
+##### `plain-di`
+
+The Device will have to configure all the required options for its
+identification with the Manufacturing server.
+
+1. `DI_MFG_STRING_TYPE`: [optional] selects the Device Identification string
```
Comment (nak3) [inline HOWTO.md:672]: This is `Device Identification` and it should be ok as it identifies the device.

**Review by nak3 — COMMENTED** [review]
This is very helpful docs! Thank you.

File: HOWTO.md (hunk 2 above)
Comment (7flying) [reply HOWTO.md:658]: yes, this one should be `Device Initialize` protocol indeed

File: HOWTO.md [hunk 4]
```diff
@@ -19,7 +19,9 @@
   - Owner Onboarding Server
   - Rendezvous Server
   - Service Info API Server
-- How to run the Client
+- How to run the clients
```
Comment (nullr0ute) [inline HOWTO.md:22]: Maybe add a : at the end here?

**Review by nullr0ute — COMMENTED** [review]
One extremely minor bit, else all good.

File: HOWTO.md (hunk 4 above)
Comment (7flying) [reply HOWTO.md:22]: will change in a sec :+1:

**Comment by 7flying** [comment]
Comments addressed and rebased on main

**Review by nullr0ute — APPROVED** [review]
LGTM!

**Pull request merged**